ce/api/stats.py

```diff
@@ -26,7 +26,12 @@
 
 
 def stats(
-    sesh, id_, time, area, variable, is_thredds=False,
+    sesh,
+    id_,
+    time,
+    variable,
+    area=None,
+    is_thredds=False,
 ):
```

Comment:
It won't mess up web calls where every argument has a parameter string in the URL, but it will break p2a's precalculation scripts. I was planning to release this as a new major version for that reason.